nbqa/__main__.py

```diff
@@ -302,24 +313,29 @@ def _preserve_config_files(nbqa_config: Optional[str], tmpdirname: str) -> None:
         Config file for third-party tool (e.g. mypy).
     tmpdirname
         Temporary directory to store converted notebooks in.
+    project_root
+        Root of repository, where .git / .hg / .nbqa.ini file is.
     """
     if nbqa_config is None:
         return
     Path(tmpdirname).joinpath(
-        Path(nbqa_config).resolve().relative_to(Path.cwd())
+        Path(project_root / nbqa_config).resolve().relative_to(project_root)
     ).parent.mkdir(parents=True, exist_ok=True)
     shutil.copy(
-        str(nbqa_config),
+        str(project_root / nbqa_config),
         str(
             Path(tmpdirname).joinpath(
-                Path(nbqa_config).resolve().relative_to(Path.cwd())
+                Path(project_root / nbqa_config).resolve().relative_to(project_root)
             )
         ),
     )
```

Comment:
This can be refactored to:
```python
temp_config_path  = Path(tmpdirname) / Path(project_root / nbqa_config).resolve().relative_to(project_root)
temp_config_path.parent.mkdir(parents=True, exist_ok=True)
shutil.copy(
        str(project_root / nbqa_config),
        str(temp_config_path )
)
```

Reply:
also a big improvement - thank you so much!